app/cluster_handlers.go

```diff
@@ -120,7 +121,11 @@ func (s *Server) clusterClearSessionCacheForAllUsersHandler(msg *model.ClusterMe
 }
 
 func (s *Server) clusterBusyStateChgHandler(msg *model.ClusterMessage) {
-	s.serverBusyStateChanged(model.ServerBusyStateFromJson(bytes.NewReader(msg.Data)))
+	var sbs model.ServerBusyState
+	if jsonErr := json.Unmarshal(msg.Data, &sbs); jsonErr != nil {
+		mlog.Warn("Failed to decode server busy state from JSON", mlog.Err(jsonErr))
```

Comment:
Should the code `return` here?(There are a lot of code pieces like this, so only asking this once)

Reply:
I've decided to keep the existing semantic and just logging to avoid unexpected results. We can revisit all these in the near future.